events-enclave-processor/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.10
+
+WORKDIR /code
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "./processor.py", "--daemonize"]
```

Comment:
Ah -- so I incorporated this when writing tests as otherwise the code loop would prevent `main()` from exiting. I updated that from our pairing session so by default the processor exits [here](https://github.com/openstax/raise-data/blob/659b07b6c460b79447d8b9857191688ec037a211/events-enclave-processor/processor.py#L145-L146) after a single pass through and only "daemonizes" itself / runs in a loop if the flag is passed. 

Note that I also left it so the loop exits if there's an unexpected exception type. I figured that will make it more clear when we deploy things if something is off since we'll immediately see the container cycling.